examples/flutter_counter/test_driver/screens/app_screen.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart';
+
+class AppScreen {
+  AppScreen(FlutterDriver flutterDriver) {
+    _driver = flutterDriver;
+  }
+
+  final incrementFloatingButton =
+      find.byValueKey('counterView_increment_floatingActionButton');
+  final decrementFloatingButton =
+      find.byValueKey('counterView_decrement_floatingActionButton');
+  final appBarText = find.text('Counter');
+  final counterState = find.byValueKey('counterState');
+  FlutterDriver _driver;
```

Comment:
Can this be final?

Reply:
```const AppScreen(this._driver);```
This cause an error that's why this can not be final.

Reply:
Done Thanks